src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js

```diff
@@ -127,6 +140,20 @@ class ActivityFeed extends React.Component<Props, State> {
         if (didLoadFeedItems || hasActiveFeedEntryIdChanged) {
             this.scrollToActiveFeedItemOrErrorMessage();
         }
+
+        // Load replies if active entry reply is set and it's not currently within feed items or its replies
+        if (
+            onShowReplies &&
+            activeFeedEntryId &&
+            activeFeedEntryType &&
+            (activeFeedEntryType === FEED_ITEM_TYPE_COMMENT || activeFeedEntryType === FEED_ITEM_TYPE_ANNOTATION) &&
+            hasLoaded &&
+            activeFeedEntryReplyId &&
+            currFeedItems &&
+            !isItemInFeed(currFeedItems, activeFeedEntryReplyId)
+        ) {
+            onShowReplies(activeFeedEntryId, activeFeedEntryType);
```

Comment:
This seems very indirect. Can we load the replies if we detect that they're needed within the `fetchFeedItems` chain?

Reply:
Yes, that was a wrong approach. I've reverted the `ActivityFeed` completely and fetching for replies is now a part of the `fetchFeedItems` chain during the first load of `ActivitySidebar`.